go/private/repositories.bzl

```diff
@@ -173,9 +173,25 @@ def go_rules_dependencies(force = False):
         patch_args = ["-p1"],
     )
 
+    # releaser:upgrade-dep grpc grpc-go
+    wrapper(
```

Comment:
Do we even need this repo now? It doesn't seem to be referenced in our BUILD files.

Reply:
Indeed, I removed it!